elbepack/schema/dbsfed.xsd

```diff
@@ -2877,6 +2877,55 @@ SPDX-FileCopyrightText: Linutronix GmbH
     </annotation>
     <simpleContent>
       <extension base="rfs:string">
+        <attribute name="origin" type="string" use="optional">
+          <annotation>
+            <documentation>
+              prefer the defined version of the debian package
+            </documentation>
+          </annotation>
+        </attribute>
+        <attribute name="release-name" type="string" use="optional">
+          <annotation>
+            <documentation>
+              prefer the defined codename of the debian package
+            </documentation>
+          </annotation>
+        </attribute>
+        <attribute name="release-archive" type="string" use="optional">
+          <annotation>
+            <documentation>
+              prefer the defined archive/suite of the debian package
+            </documentation>
+          </annotation>
+        </attribute>
+        <attribute name="release-version" type="string" use="optional">
+          <annotation>
+            <documentation>
+              prefer the defined Debian release version of the debian package
+            </documentation>
+          </annotation>
+        </attribute>
+        <attribute name="release-component" type="string" use="optional">
+          <annotation>
+            <documentation>
+              prefer the defined component of the debian package
+            </documentation>
+          </annotation>
+        </attribute>
+        <attribute name="release-origin" type="string" use="optional">
+          <annotation>
+            <documentation>
+              prefer the defined originator of the debian package
+            </documentation>
+          </annotation>
+        </attribute>
+        <attribute name="release-label" type="string" use="optional">
+          <annotation>
+            <documentation>
+              prefer the defined label of the debian package
+            </documentation>
+          </annotation>
+        </attribute>
         <attribute name="pin" type="string" use="optional">
```

Comment:
Just add it to `<documentation>`: Deprecated, use attribute "release-name" instead.
If we have many of those this could be formalized but that will also need changes to the documentation generation.